client-js/app/elements/labrad-registry.ts

```diff
@@ -129,6 +129,101 @@ export class LabradRegistry extends polymer.Base {
     );
   }
 
+  //Drag and drop
+  @listen('dragenter')
+  overBehaviour(event) {
+    event.preventDefault(); //I don't understand this.
+  }
+
+  @listen('dragover')
+  changeBehaviour(event) {
```

Comment:
I would recommend naming all these methods after the events they listen to in a consistent way, e.g.
```typescript
@listen('dragenter')
onDragEnter(event) { ... }

@listen('dragover')
onDragOver(event) { ... }

...
```